corehq/apps/app_manager/templates/app_manager/partials/app-settings.html

```diff
@@ -151,51 +164,95 @@
 
 <div class="tab-pane" id="commcare-settings">
     <div class="clearfix" id="settings-save-btn" data-bind="saveButton2: state, saveOptions: saveOptions"></div>
-    <form class="form-horizontal" data-bind="foreach: sections">
-        <fieldset data-bind="visible: notEmpty">
+    <form class="form-horizontal">
```

Comment:
Thanks for the heads up. Easier to read with https://github.com/dimagi/commcare-hq/pull/5858/files?w=1 I think